peachjam_search/templates/peachjam_search/search.html

```diff
@@ -6,6 +6,11 @@
 {% block page-content %}
   <div>
     <div class="container pt-lg-5 pt-2 pb-5">
+      <p class="lead text-center">
+        {% blocktrans trimmed %}
+        Search regional and national African legal information.
```

Comment:
This will need to be done in the africanlii app so that other places don't pick it up.

Reply:
@longhotsummer This view is served by the peachjam_search app not the main peachjam app. Whats the best way to override it?

Reply:
Create a file called `africanlii/templates/peachjam_search/search.html` and it will override this one, like normal. You'll need to setup a block in the base template to override it easily in the new template.

Reply:
:sweat_smile:  I forgot that you can simply subclass a template, I was thinking I had to pull the search app into africanlii